pkg/reconciler/reconciler_test.go

```diff
@@ -1545,6 +1509,135 @@ var _ = Describe("Reconciler", func() {
 		})
 	})
 
+	_ = Describe("WithSelector test", func() {
```

Comment:
For consistency, since most other funcs do not mention "test" in the name.

```suggestion
	_ = Describe("WithSelector", func() {
```